index.js

```diff
@@ -1,16 +1,18 @@
-"use strict"
-var os = require("os")
+"use strict";
+const os = require("os");
 
-var hasUnicode = module.exports = function () {
+const hasUnicode = module.exports = function () {
```

Comment:
I'm not sure there's any value in updating this module ever, but if I were going to, which to const is an appropriate change. But I would rewrite this line to be `module.exports = function hasUnicode () {` as nothing uses the hasUnicode name here, and this way the exported function has its name bound to it.